google-cloud-tools-plugin/ultimate/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2000-2016 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+sourceSets.main.resources.srcDirs = ['resources']
+sourceSets.main.java.srcDirs = ['src']
+sourceSets.test.java.srcDirs = ['testSrc']
+
+intellij {
+    type 'IU'
+
+    //actually only 'JavaEE' and 'GwtStudio' plugins are required, the others are transitive dependencies
```

Comment:
I think (its another thing we inherited) it means that only the JavaEE and GwtStudio (although I'm not sure about this one) are used directly by the plugin. The other ones are transitive dependencies of two that need to be explicitly defined.